tools/release-plugin.sh

```diff
@@ -67,30 +56,67 @@ while getopts "v:abh" opt; do
 done
 shift "$(($OPTIND -1))"
 
-# If there are more than two arguments, bail.
-[[ $# -gt 2 ]] && die "Too many arguments specified! Only provide a project and a version number. Got:$(printf ' "%s"' "$@")"$'\n'"(note all options must come before the project slug)"
+# Parse arguments in associated array of plugins/project => version format.
+if [ $# -eq 0 ]; then
+	usage
+fi
+
+declare -A PROJECTS
+while [[ $# -gt 0 ]]; do
+	if [[ "$1" =~ ^[a-zA-Z\-]+$ ]]; then
+		PLUGIN=$1
+	else
+		die "Script must be passed in [project] <version> format. Got $1"
```

Comment:
The usual convention is to use `[]` to delimit optional arguments, so I'd do like
```suggestion
		die "Script must be passed in <project> [<version>] format. Got $1"
```
or maybe
```suggestion
		die "Script must be passed in <project> [version] format. Got $1"
```

Reply:
Oh! I had them backwards, thank you.